### ISSUE TITLE
absolute value of depth for a specific object in high resolution images and without any GT data

### ISSUE THREAD

**Issue by hlmhlr** [JiawangBian/sc_depth_pl#50]
Hi, 

The work is really appreciated!! Good job. 

I have the following basic queries:

i. I am trying to generate the depth images of a custom dataset using given pre-trained models. Images have 4K resolution while models are trained at small resolutions (e.g 320x256 in bonn dataset). For a single 4k image, its respective depth image size is 320x256. To have the depth value for the original 4k images, I should calculate the value of depth from a small size depth image and then simply plug it with the original image or there is any way around it?

ii. Since I have a custom dataset without any Ground Truth (GT) data, after predicting the depth images, how can I calculate the absolute value of depth (in meters) for a particular object in an image? I have the maximum value of depth of the room where the camera is installed.

**Comment by JiawangBian** [comment]
1. Current model cannot predict depth for 4K resolution. What you can do here is just up-sampling. You may train the model on higher-resolution images if you really need high resolution. 
2. You can roughly recover the metric scale by aligning the maximum depth value, but it is inaccurate. It depends on your requirements.
